src/libraries/System.Net.WebSockets.Client/tests/CloseTest.cs

```diff
@@ -69,9 +69,10 @@ public async Task CloseAsync_ServerInitiatedClose_Success(Uri server, bool useCl
 
                 // Send back close message to acknowledge server-initiated close.
                 _output.WriteLine("Close starting.");
+                var closeStatus = PlatformDetection.IsNotBrowser ? WebSocketCloseStatus.InvalidMessageType : (WebSocketCloseStatus)3210;
```

Comment:
is the number 3210 important?

Reply:
It needs to be in range that the browser can send. See https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/close#code